specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/registries.json

```diff
@@ -325,6 +325,19 @@
             "schema": {
               "$ref": "#/definitions/RegistryTrackedResource"
             }
+          },
+          "202": {
```

Comment:
>         "202": { [](http://example.com/codeflow?start=1&length=17)

put must not return 202

Reply:
Our service returns a 202 on put here  (adding this in so that the sdk client doesn't break per this issue: https://github.com/Azure/azure-rest-api-specs/issues/25119) What would you recommend be done here/ what should the service return instead of 202 for LRO PUT operations?

Reply:
Service should return 201 for Async Put requests, which you have already have in this swagger, so change the implementation, if needed. And, remove 202 from here.